app/main.py

```diff
@@ -1,5 +1,6 @@
 def copy_file(command: str) -> None:
-    com, old_file, new_file = command.split()
+    split_command = command.split()
+    old_file, new_file = split_command[1], split_command[2]
```

Comment:
you can do this in on line
```suggestion
    old_file, new_file = command.split()
```

Reply:
I can't do this, because in command I have string _"cp file.txt file.txt"_